### ISSUE TITLE
[Suggestion] Biomes O' Plenty compatibility

### ISSUE THREAD

**Issue by HexabluDEV** [CheezedFish/SpartanCompatibility#9]
hello! didn't have a github account yet so i made one so i could leave this request, i'm sorry if i'm not doing this right as it is my first time.
 
My suggestion would be to add support for the Biomes O' Plenty gemstones: 
Ruby, Peridot, Topaz, Tanzanite, Malachite and Amber!

thanks for making this mod, its amazing!

**Comment by itzjrdn** [comment]
yes this would be very good i hope he adds this is is probably the most needed one
